csfieldguide/static/interactives/binary-cards/js/binary-cards.js

```diff
@@ -26,18 +38,57 @@ $(document).ready(function () {
         updateDotCount();
     });
 
+    // Update number of cards shown based on input
+    $('input').change(function() {
```

Comment:
Should text input default to 1?
Maybe if input is negative try the absolute value first